Dockerfile

```diff
@@ -40,10 +40,11 @@ ARG ADDITIONAL_PYTHON_DEPS=""
 
 ARG AIRFLOW_HOME=/opt/airflow
 ARG AIRFLOW_UID="50000"
+ARG AIRFLOW_USER_HOME_DIR=/home/airflow
```

Comment:
Why do we need to add this up here? It doesn't seem to be used until the stage that (re-)defines it.

Reply:
Laked from the following change.